docs/source/plugins/06_solver_hooks.rst

```diff
@@ -174,8 +174,8 @@ Internal deposit layer
 ----------------------
 
 When a new phase is added to the hydrodynamic model using the :py:class:`~alfasim_sdk.AddPhase` type, it is possible
-to consider the deposition inside the pipeline walls. If so, calculate the thickness of the deposited layer on a given
-phase through the usage of :py:func:`~alfasim_sdk._internal.hook_specs.update_internal_deposition_layer`. By default, the layer
+to consider the deposition inside the pipeline walls. In this case, the thickness of the deposited layer on a given
+phase is calculated through the usage of :py:func:`~alfasim_sdk._internal.hook_specs.update_internal_deposition_layer`. By default, the layer
```

Comment:
```suggestion
to consider the deposition inside the pipeline walls. In this case, the thickness of the deposited layer on a given
phase should be calculated through the usage of :py:func:`~alfasim_sdk._internal.hook_specs.update_internal_deposition_layer`. By default, the layer
```

Proposing this change, because `is calculated` gives the idea that the deposition layer is calculated by alfasim instead of the user calculating it and imposing the value in the hook

Reply:
I agree! Done.